src/mysky/index.ts

```diff
@@ -774,18 +810,76 @@ export class MySky {
     await this.addPermissions(...perms);
   }
 
+  /**
+   * Handles the after-login logic.
+   *
+   * @param loggedIn - Whether the login was successful.
+   */
   protected async handleLogin(loggedIn: boolean): Promise<void> {
-    if (loggedIn) {
-      await Promise.all(
-        this.dacs.map(async (dac) => {
-          try {
-            await dac.onUserLogin();
-          } catch (error) {
-            // Don't throw on error, just print a console warning.
-            console.warn(error);
-          }
-        })
+    if (!loggedIn) {
+      return;
+    }
+
+    // Call the `onUserLogin` hook for all DACs.
+    await Promise.all(
```

Comment:
I don't think you want to use `Promise.all` here. If one DAC immediately rejects it could make it so `onUserLogin` is not called on other DACs. I think we want to use `.allSettled` here.

Reply:
Oh lol never mind, I just saw you wrap the call in a `try-catch` anyway. I'll leave the comment in the off-chance you were unaware of the behavior of `Promise.all`, but feel free to resolve it.